packages/flutter/lib/src/widgets/form.dart

```diff
@@ -278,8 +282,9 @@ class FormFieldState<T> extends State<FormField<T>> {
   bool get hasError => _errorText != null;
 
   /// Calls the [FormField]'s onSaved method with the current value.
+  /// If the [FormField]'s enabled is false, the method does nothing.
   void save() {
-    if (widget.onSaved != null)
+    if (widget.onSaved != null && widget.enabled)
       widget.onSaved(value);
```

Comment:
why do we want this to do nothing when the field is disabled? Is it valid to call this when it's disabled? If so, why, since it does nothing? If not, then we should just assert.

Reply:
It seems pointless because if a field is disabled then the user can't edit it so there's nothing to save. But maybe I'm missing something? Can you think of a case where you'd want to save the values from a disabled field?